app/main/controllers/activity/RTMediaBuddyPressActivity.php

```diff
@@ -621,17 +621,18 @@ public function bp_activity_posted_update( $content, $user_id, $activity_id ) {
 		$rtmedia_privacy = filter_input( INPUT_POST, 'rtmedia-privacy', FILTER_SANITIZE_NUMBER_INT );
 
 		if ( null !== $rtmedia_privacy ) {
-			$privacy = -1;
+
+			// Set Privacy 0 if media privacy is disabled.
+			$privacy = get_rtmedia_default_privacy();
```

Comment:
@vaishaliagola27 - Is this tested with other scenarios like mention in https://github.com/rtCamp/rtMedia/pull/1415 issue?

Reply:
@abhijitrakas yes. I've tested all those scenarios.